codecs/cpp.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM emscripten/emsdk:1.39.19
+FROM emscripten/emsdk:1.40.0
```

Comment:
I did update to Emscripten 1.40 because libavif doesn’t build with 1.39.19, but I haven’t re-built all the other codecs.

Reply:
Could you update & rebuild codecs in separate PR perhaps?

Reply:
That’s the plan